README.md

```diff
@@ -33,10 +33,13 @@ This module provides SAML2 SSO functionality for FOLIO.
 
 Endpoints are documented in [RAML file](ramls/saml-login.raml)
 
-### Environment variables
+### Environment variables and system properties
 
-`TRUST_ALL_CERTIFICATES`: if value is `true` then HTTPS certificates not checked. This is a security issue in
-production environment, use it for testing only! Default value is `false`.
+`TRUST_ALL_CERTIFICATES`: if value is `true` then HTTPS certificates not checked. This is a security issue in production environment, use it for testing only! Default value is `false`.
+
+`SAML_COOKIE_SAMESITE`: Set to `Lax` if domain for front end and backend are the same for a more secure cookie (default value `None` if this environment variable is not set).
+
+`Lax` should only be used if the backend and frontend hosts are the same, otherwise the browser will reject the cookies. There is a corresponding system property `saml.cookie.samesite`.
```

Comment:
Ok I agree we could make `Strict` the default. This will be different than what we released for mod-login, which has the configuration I have. Let me know if you think that is ok.

Reply:
Ok I made the change. The meaning of `Lax` may be somewhat [different that what you have](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies). I added slightly different language.